src/pages/about.tsx

```diff
@@ -1,4 +1,7 @@
- function about(){
-    return <h1>about page</h1>
+import { NextPage } from "next";
+const about: NextPage<{}> = () => {
```

Comment:
in thise part pleae remove the complex part and add simple form of the code

Reply:
I have add a comment on line 2 of about page for clirification

Reply:
ik thank you it is approved and merge it